db/seeds.rb

```diff
@@ -6,8 +6,8 @@
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
 
-ashby = User.create username: "ashby", email: Rails.application.credentials.admin_email
-ashby.create_identity(
+User.create(
+  username: "ashby",
   email: Rails.application.credentials.admin_email,
   password: Rails.application.credentials.admin_password,
```

Comment:
> Rails.application.credentials.admin_password

hm?

Reply:
https://github.com/rails/rails/pull/30067 It's neat

Reply:
So thats where Ambers encrypted secrets file idea came from...